cmd/web/config/config_test.go

```diff
@@ -532,22 +535,24 @@ func TestString_GetSensored(t *testing.T) {
 	cfg := Config{}
```

Comment:
Test name doesn't match the implementation, test should be named `TestConfig_String` (Since it's a `Test` for the  type `Config` with (separated with a `_` for readability I guess) the method `String`